lib/widgets/home.dart

```diff
@@ -141,28 +95,38 @@ class _HomeState extends State<Home> {
                 mainAxisAlignment:
                     MainAxisAlignment.end, //align the button to the right side
                 children: [
-                  IconButton(
-                      //button initialisation
-                      icon: Icon(
-                        Icons.settings,
-                        color: Colors.black54,
-                      ),
-                      onPressed: () {
-                        // this is what happens when the settings button is pressed
-                        Navigator.push(
-                          context,
-                          MaterialPageRoute(
-                              builder: (context) =>
-                                  SettingsPage()), // We use the Navigator to Route to the settings page wich is located in a new .dart file
-                        );
-                      }),
+                  customShowCase(
+                    settings,
+                    "Einstellungen",
+                    "Klicken Sie hier, um zu den Einstellungen zu gelangen",
+                    IconButton(
+                        //button initialisation
+                        icon: Icon(
+                          Icons.settings,
+                          color: Colors.black54,
+                        ),
+                        onPressed: () {
+                          // this is what happens when the settings button is pressed
```

Comment:
bestes kommentar... besser: when settings button is pressed, route to settings page